tools/ci/azure/install_edge.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  channel: canary
+
+# Should match https://web-platform-tests.org/running-tests/safari.html
```

Comment:
Oops, this must be copied from another file. But documenting for users for WPT how to run the different versions of Edge in https://github.com/web-platform-tests/wpt/tree/master/docs/running-tests would be nice.

Reply:
Good catch. I updated PR to point to chrome.md file since at this point we support same flags.